packages/headless/src/features/analytics/insight-analytics-actions.test.ts

```diff
@@ -101,3 +108,39 @@ describe('logInterfaceChange', () => {
     );
   });
 });
+
+describe('the analytics related to the create article feature in the insight use case', () => {
+  const engine = buildMockInsightEngine({
+    state: buildMockInsightState({
+      insightCaseContext: {
+        caseContext: {
+          Case_Subject: exampleSubject,
+          Case_Description: exampleDescription,
+        },
+        caseId: exampleCaseId,
+        caseNumber: exampleCaseNumber,
+      },
+    }),
+  });
+
+  it('should log #logCreateArticle with the right payload', async () => {
+    await engine.dispatch(
+      logInsightCreateArticle(exampleCreateArticleMetadata)
+    );
+    const expectedCaseContext = {
+      caseContext: {
+        Case_Subject: exampleSubject,
+        Case_Description: exampleDescription,
+      },
+      caseId: exampleCaseId,
+      caseNumber: exampleCaseNumber,
+    };
+    expect(mockLogCreateArticle).toHaveBeenCalledTimes(1);
```

Comment:
In the other two tests of this file we are still using `toBeCalledTimes` which is I think is deprecated, could you please replace `toBeCalledTimes` by `toHaveBeenCalledTimes` since we are already here modifying this file.